virtualevent-demo/build.gradle

```diff
@@ -1,7 +1,6 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
-    id 'androidx.navigation.safeargs.kotlin'
 }
```

Comment:
No longer needed as we only use jetpack navigation for switching between tabs. No args are passed to them.